description: restore support for building with make rather than ninja

--- a/src/build/gyp_chromium
+++ a/src/build/gyp_chromium
@@ -246,12 +246,6 @@ if __name__ == '__main__':
   if sys.platform not in ('darwin',):
     args.append('--no-circular-check')
 
-  # We explicitly don't support the make gyp generator (crbug.com/348686). Be
-  # nice and fail here, rather than choking in gyp.
-  if 'make' in os.environ.get('GYP_GENERATORS', ''):
-    print 'Error: make gyp generator not supported (check GYP_GENERATORS).'
-    sys.exit(1)
-
   # Default to ninja on linux and windows, but only if no generator has
   # explicitly been set.
   # Also default to ninja on mac, but only when not building chrome/ios.
